From ab0d35e930bc76a5fde883021f4189f0202ef954 Mon Sep 17 00:00:00 2001
From: Jason Hung <jhung@globalscaletechnologies.com>
Date: Fri, 22 Nov 2024 10:43:51 +0800
Subject: [PATCH] dts: gti-mt510ss-evk: change uart2 pins to GPIO [79-80] for
 hw v1.1.0

since hw rev 1-1-0 change the uart2 tx/rx pins to gpio 79/80.
for this hw version, we can use rs232 and rs485 at the same time.
---
 arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts b/arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts
index 9be16b831377..dd127e3c526c 100644
--- a/arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts
+++ b/arch/arm64/boot/dts/mediatek/gti-mt510ss-evk.dts
@@ -597,8 +597,8 @@ pins_cts {
 
 	uart2_pins: uart2-pins {
 		pins {
-			pinmux = <PINMUX_GPIO35__FUNC_O_UTXD2>,
-				 <PINMUX_GPIO36__FUNC_I1_URXD2>;
+			pinmux = <PINMUX_GPIO79__FUNC_O_UTXD2>,
+				 <PINMUX_GPIO80__FUNC_I1_URXD2>;
 			bias-pull-up;
 		};
 	};
@@ -1117,7 +1117,7 @@ &uart1 {
 &uart2 {
 	pinctrl-0 = <&uart2_pins>;
 	pinctrl-names = "default";
-	status = "disabled";
+	status = "okay";
 };
 
 &mipi_tx0 {
-- 
2.34.1

